.github/workflows/haskell.yml

```diff
@@ -75,6 +75,7 @@ jobs:
             ghc_version: 8.6.5
             runner: ubuntu-latest
             cabal_version: 3.2
+            pack_options: DISABLE_TEST=n
```

Comment:
Comment out the DISABLE_TEST=y in the global settings and use DISABLE_TEST=y in tests where you want to disable it. Default should be enabled rather disabled.